hm.tex

```diff
@@ -1334,12 +1336,12 @@ \subsection{Back to one degree of freedom, again}\label{sec:1deg-again}
 \begin{equation}
     x(t) = \left( R_1 \cos(t + \phi_1),\, R_2 \cos(\omega t + \phi_2) \right).
 \end{equation}
-This is contained in a rectangle $[-R_1,R_1]\times[-R_2,R_2]$. When $\omega$ is irrational, the trajectory fills the rectangle while they are closed curves inside the rectangle when it is rational.
+This is contained in a rectangle $[-R_1,R_1]\times[-R_2,R_2]$. When $\omega$ is irrational, the trajectory fills the rectangle, whereas they are closed curves inside the rectangle when it is rational. % not always closed, the top right example in the figure corresponds to a rational omega but it is not closed.
```

Comment:
It is a degenerate ellipse squeezed to a line, it does not really have endpoints. Maybe this could be mentioned in the caption, what do you think?

```suggestion
This is contained in a rectangle $[-R_1,R_1]\times[-R_2,R_2]$. When $\omega$ is irrational, the trajectory fills the rectangle, whereas they are closed curves inside the rectangle when it is rational.
```

Reply:
I think it's a good idea to put it in a comment, I'll do that.